libpod/runtime.go

```diff
@@ -172,6 +172,8 @@ type RuntimeConfig struct {
 	// However, this can cause significant memory usage if a container has
 	// many ports forwarded to it. Disabling this can save memory.
 	EnablePortReservation bool `toml:"enable_port_reservation"`
+	//LabelSupport indicates wether libpod will support container labeling
```

Comment:
Missing a space before LabelSupport.

Reply:
Also - maybe rename this to EnableLabelling?

Reply:
Fixed, although when with EnableLabeling.